docs/build/tooling/hardhat/hardhat-zksync-upgradable.md

```diff
@@ -212,6 +212,29 @@ npx hardhat run SCRIPT_FILE
 - This provider is configured in the hardhat config file, by stating the RPC url of the network to connect to.
   :::
 
+Additionally, in the options section optionally include the folowing arguments to configure the deployment of the proxy and implementation with different deployment types and salts:
+
+- `deploymentTypeImpl`
+- `saltImpl`
+- `deploymentTypeProxy`
+- `saltProxy`
+
+```
+ const box = await hre.zkUpgrades.deployProxy(deployer.zkWallet, contract, [42],
+  { initializer: "initialize",
+    saltImpl: "0x4273795673417857416686492163276941983664248508133571812215241323",
+    deploymentTypeImpl: "create2",
+    saltProxy: "0x5273795673417857416686492163276941983664248508133571812215241323",
+    deploymentTypeProxy: "create2"
+  }
+);
+```
+
+:::note
+Permissible values for the deployment type include `create`, `create2`, `createAccount`, and `create2Account`. If this parameter is omitted, the default value will be `create`.
+If the salt parameters are omitted, the default value will be `0x0000000000000000000000000000000000000000000000000000000000000000`.
+:::
+
 ### Openzeppelin Version
 
 The plugin does not work with the latest versions due to a blocker on the `@matterlab/zksync-contracts` package.The solution is to change the development dependencies to the previous version in your `package.json`.
```

Comment:
Can we remove this section and merge it with the warning in the "Installation" section at the beginning of the page?